octobot/src/server/github_handler.rs

```diff
@@ -406,6 +407,25 @@ impl GithubEventHandler {
         let mut participants = pull_request.assignees();
         // add the author of the PR
         participants.push(pull_request.user().clone());
+        // add team members
+        let teams = pull_request.teams();
+        let futures = teams.iter().map(|t| {
+            self.github_session
+                .get_team_members(t.org(), t.slug.as_str())
+        });
+        let teams = join_all(futures).await;
+        let mut members: Vec<User> = vec![];
```

Comment:
i think you probably can ditch `members` since the thing you really want to append to is `participants`.